base/vulkanexamplebase.h

```diff
@@ -50,9 +50,59 @@
 #define GAMEPAD_BUTTON_R1 0x1005
 #define GAMEPAD_BUTTON_START 0x1006
 
+#if defined(__linux__) && !defined(__ANDROID__)
```

Comment:
I think I'd prefer Android and Linux to be separate branches, even if one is currently empty. E.g.:
```
#if defined(__ANDROID__)
#elif defined(__linux__)
#define ...
```
This makes the behaviour of an `#else` branch more predictable *(currently, Android, Windows and anything else fall in that `#else` on line 84)*, and makes it easier to add Android-specific code later.